### ISSUE TITLE
cheroot python3 fix setup patch changed?

### ISSUE THREAD

**Issue by dylex** [NixOS/nixpkgs#47320]
## Issue description

```
trying https://git.archlinux.org/svntogit/community.git/plain/trunk/cheroot-fix-setup-python3.patch?h=packages/python-cheroot
fixed-output derivation produced path '/mnt/home/sw/nix/store/ad4q6fzgzqwx4h3nbasgb0whm9w2rpfg-cheroot-fix-setup-python3.patch' with sha256 hash '0wy46ra8kb4rd9z1gy0xczhgpkqkf9r1gfs223lxl89c8a9l33b0' instead of the expected hash '1rlgz0qln536y00mfqlf0i9hz3f53id73wh47cg5q2vcsw1w2bpc'
```

### Steps to reproduce

Build python3.pkgs.cheroot.

The patch URL in pkgs/development/python-modules/cheroot/default.nix should maybe refer to a specific commit?
https://git.archlinux.org/svntogit/community.git/plain/trunk/cheroot-fix-setup-python3.patch?h=packages/python-cheroot

**Comment by dylex** [comment]
This URL seems to work to get the old patch, if that's the intention:
https://git.archlinux.org/svntogit/community.git/plain/trunk/cheroot-fix-setup-python3.patch?h=packages/python-cheroot&id=9b33cb0885b3c0d91adeacae23761a4321eb0e64

**Comment by dylex** [comment]
Fixed by #47354, assuming the desire is to continue using the old version of the patch.
